### ISSUE TITLE
add server "Description" field as an alias to long/similar addresses and ips

### ISSUE THREAD

**Issue by akempgen** [colloquy/colloquy#1295]
*Issue migrated from trac ticket [#1070](http://colloquy.info/project/ticket/1070)*

**type:** Enhancement | **component:** Colloquy (Mac) | **severity:** Normal | **priority:** Low | **version:** 2.1 (Mac) | **status:** assigned
___
#### 2007-07-19 09:32:04: ScottM created the issue

It would be handy if the Connection Panel could accept some sort of "alias" for a server rather than having to display the full server name.  I am on loads of servers, some of which have long and annoying names, and simply being able to refer to it by a shorthand alias would be a nice visual enhancement.

For example: 

Freenode
Undernet
Blorp7

And so on.

Thanks.

___
#### 2007-07-21 00:49:21: @xeenon changed status from *new* to *assigned*

___
#### 2009-04-17 04:18:57: @akempgen changed title from *Aliases for server names in the connection panel rather than full hostname/IP* to *add server "Description" field as an alias to long/similar addresses and ips*

___
#### 2009-04-17 04:18:57: @akempgen commented

basically the same thing that mobile already has. irc networks offer a "NETWORK=QuakeNet" in 005, maybe that could be used as a fallback

___
#### 2009-04-27 04:32:31: @zadr changed owner from *timothy* to *zach*

___
#### 2009-04-27 04:32:31: @zadr changed severity from *Minor* to *Normal*

___
#### 2013-02-18 23:19:33: CyberJacob <cyberjacob@… commented

Sorry for pulling this one out of the grave, but is there any progress on implementing it?

___
#### 2013-02-23 23:07:20: @zadr commented

Its still on the todo list. Not much movement on it however, sorry.

**Comment by Ferrisx4** [comment]
This would be tremendously useful for people who connect to several bouncer accounts hosted at the same address, especially when those bouncer accounts have different usernames and passwords. (This sort of situation occurs with bouncers such as ZNC that only connect to one IRC server per account). 
Colloquy currently likes to keep the same server password for every server entry that has the same server so if two of my bouncer accounts have different passwords, I have to be use different server addresses to reach it. For instance—foo.com, irc.foo.com, 123.456.789.111. Otherwise they collide and Colloquy gets confused. 

Sidenote: the Affected tag should also be updated to include more versions than 2.1.
I know this is ancient.

**Comment by zadr** [comment]
fwiw the affected tag indicates the first affected version at the time of the creation of the ticket, not all affected versions.

**Comment by nickurak** [comment]
Basic workflows are pretty tricky with multiple servers that have the same hostname -- eg, the join dialog will ask which server you want to join a channel on, and they'll all be indistinguishable.
